docs/ios-manual-logging.md

```diff
@@ -4,12 +4,13 @@ title: Scope iOS Agent manual logging
 sidebar_label: Manual logging
 ---
 
-In order to improve the information you can get from Scope, you can also add your custom application logs. 
-It will allow you to contextualize much better the state of your application when you review the Scope output.
+Scope by default otputs `NSLog`, `print` and `os_log` (stdout and stderr) messages in the Logs, but sometimes those messages are associated to the test Span instead of being associated to the Span where the log was generated. 
```

Comment:
```suggestion
Scope captures `NSLog`, `print` and `os_log` (stdout and stderr) output automatically, and associates them to the active span when they were generated if possible, or to the test span otherwise.
```